waiter/test-files/grpc/grpc_client_exit.clj

```diff
@@ -0,0 +1,64 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns grpc.grpc-client-exit
+  (:require [clj-time.core :as t]
+            [clj-time.format :as f]
+            [clojure.test :refer :all]
+            [waiter.util.client-tools :refer :all]
+            [waiter.util.date-utils :as du]
+            [waiter.util.utils :as utils])
+  (:import (com.twosigma.waiter.courier GrpcClient GrpcClient$CancellationPolicy)
+           (java.util.function Function)))
+
+(defn- initialize-grpc-client
+  "Initializes grpc client logging to specific correlation id"
+  [correlation-id host port]
+  (let [date-formatter (:date-hour-minute-second-fraction f/formatters)
+        log-function (reify Function
+                       (apply [_ message]
+                         (println (du/date-to-str (t/now) date-formatter) correlation-id message)))]
+    (GrpcClient. host port log-function)))
+
+(try
```

Comment:
Is this a command line utility?

Reply:
Yes, this is invoked from the test in a separate process as it will exit the JVM.